tests/test_predefined.py

```diff
@@ -33,7 +33,7 @@ def test_DNVGL_Hs_Tz():
     # Compare with the results from contours presented in Haselsteiner et al. (2019),
     # https://doi.org/10.1115/OMAE2019-96523
     # Highest Hs value is ca. 5 m, highest Tz value is ca. 16.2 s.
```

Comment:
We changed the predefined model to use the MoM so the expected results are different.
```suggestion
    # The highest Hs value is ca. 10 m and the highest Tz value is ca. 17 s,
    # see https://github.com/virocon-organization/virocon/pull/228/files#r2007155949
```